.ci/pipeline_definitions

```diff
@@ -47,19 +48,6 @@ cc-utils:
             - '--package'
             - 'gardener-cicd-cli'
       traits:
-        component_descriptor:
-          retention_policy:
-            name: 'cleanup-snapshots-and-releases'
-            dry_run: False
-            rules:
-              - name: 'snapshots'
-                match: 'snapshots'
-                restrict: 'none'
-                keep: 15
-              - name: 'releases'
```

Comment:
Since there is only an automated cleanup for snapshots repository configured, shouldn't we keep the retention policy for releases?

Reply:
hmmm... yeah, I suppose so

Reply:
I moved cleanup to release-job, though (there is no need to run it more often, is there..)